webpack.config.js

```diff
@@ -2,8 +2,8 @@ const path = require('path');
 const StyleLintPlugin = require('stylelint-webpack-plugin');
 const VisualizerPlugin = require('webpack-visualizer-plugin');
 
-const MAX_DEVELOPMENT_OUTPUT_SIZE = 1850000;
-const MAX_PRODUCTION_OUTPUT_SIZE = 210000;
+const MAX_DEVELOPMENT_OUTPUT_SIZE = 2000000;
```

Comment:
Why the size increse? This PR should have a negative imapct:

![Screenshot from 2021-02-26 14-21-56](https://user-images.githubusercontent.com/4589176/109305488-14f77800-783e-11eb-91cd-f89580a1e096.png)

Reply:
It's because of new theming options for refactored components. Until now, the design of `CheckboxField`, `Radio` and `Toggle` was hard-coded to meet the requirements of _our project_. Now the default appearance is less opinionated while it's still perfectly possible to get back to the original design through theming options, just like with any other component.

However, I believe there is a [chance to make the bundle slightly smaller](https://github.com/react-ui-org/react-ui/issues/236) (but not drastically).

Reply:
The new version has less lines of code yet it produces a larger package :thinking:

Reply:
Less source SCSS, more output CSS.